src/command/cmd_funcs.c

```diff
@@ -5091,6 +5091,10 @@ cmd_alias(ProfWin* window, const char* const command, gchar** args)
             cons_bad_cmd_usage(command);
             return TRUE;
         } else {
+            if (strchr(alias, ' ')) {
```

Comment:
Actually parse_args_with_freetext() and not parse_args() is used. So that's why it won't be split up by spaces.
And indeed I can do `/alias add "afk asdf" /status set away \"Away From Keyboard\"` to have a space in the alias. And with your check you disallow this. I see!